common/src/main/java/com/microsoft/identity/common/internal/request/MsalBrokerRequestAdapter.java

```diff
@@ -99,9 +106,9 @@ public BrokerRequest brokerRequestFromAcquireTokenParameters(@NonNull final Acqu
                         parameters.getExtraQueryStringParameters() != null ?
                                 QueryParamsAdapter._toJson(parameters.getExtraQueryStringParameters())
                                 : null
-                ).prompt(parameters.getOpenIdConnectPromptParameter().name())
+                ).prompt(parameters.getPrompt().name())
                 .claims(parameters.getClaimsRequestJson())
-                .forceRefresh(parameters.getForceRefresh())
+                .forceRefresh(false)
```

Comment:
Interactive doesn't have anything like `forceRefresh`

Reply:
This would break `claimsRequest` flow. Before this change , `forceRefresh` was on `OperationParameter` [here ](https://github.com/AzureAD/microsoft-authentication-library-common-for-android/blob/dev/common/src/main/java/com/microsoft/identity/common/internal/request/OperationParameters.java#L85) and we set `forceRefresh` to true for interactive calls when `claimsRequest` is present [here ](https://github.com/AzureAD/microsoft-authentication-library-for-android/blob/dev/msal/src/main/java/com/microsoft/identity/client/internal/controllers/OperationParametersAdapter.java#L147). This is done because, in broker we would attempt to silently get a token for interactive calls as well if we a matching account using PRT, FRT, and if `claimsRequest` is present we would need to skip cache and refresh the token.

So i think you should move back `forceRefresh` to base class and add the logic back